interceptor_ext_test.go

```diff
@@ -123,7 +123,7 @@ func TestOnionOrderingEndToEnd(t *testing.T) {
 	mux := http.NewServeMux()
 	mux.Handle(
 		pingv1connect.NewPingServiceHandler(
-			pingServer{},
+			pingServerGenerics{},
```

Comment:
This is because I had to create a new `pingServer` that uses the simple API also for tests. Since I figure we want `pingServer` to be the simple API, i instead renamed this one to `pingServerGenerics`.